lib/site_encrypt.ex

```diff
@@ -47,6 +47,13 @@ defmodule SiteEncrypt do
   @doc "Invoked after the new certificate has been obtained."
   @callback handle_new_cert() :: any
 
+  log_levels =
+    if Version.compare(System.version(), "1.11.0") in [:gt, :eq] do
+      [:error, :info, :debug, :emergency, :alert, :critical, :warning, :notice]
+    else
+      [:debug, :info, :warn, :error]
+    end
+
```

Comment:
I don't think there's need for these version gymnastics. I propose we use `Logger.levels/0`, but we should then change mix.exs and require 1.16+. WDYT?

Reply:
I'll update the PR in the next few minutes.

Reply:
https://github.com/sasa1977/site_encrypt/pull/63/commits/2e804e6eb88a9c74dbcd33be1d94aae81bae44ae